src/N98/Magento/Command/Developer/Module/Rewrite/ConflictsCommand.php

```diff
@@ -167,6 +171,45 @@ protected function _isInheritanceConflict($classes)
     }
 
     /**
+     * Check if one of the rewritten classes actually is written to fix the conflict.
+     * We can check by checking for an annotation in the PHPDOC block (@SolvesRewriteConflict)
+     *
+     * @var array $rewriteClasses
+     * @var string $rewrittenClass
+     * @return bool
+     */
+    protected function _isARewriteFixingTheConflict($rewriteClasses, $rewrittenClass)
```

Comment:
For new code we try to make the following differences:

# Prefer private over protected - do as little as necessary, this includes reduce visibility to the bare minimum.

# No underscore-prefix for method names - This is an outdated, very-old-PHP-4-style to signal method visibility. This is not needed any longer since PHP 5 nor recommended.